example/example/settings.py

```diff
@@ -46,10 +46,6 @@
 # to load the internationalization machinery.
 USE_I18N = True
 
-# If you set this to False, Django will not format dates, numbers and
-# calendars according to the current locale.
-USE_L10N = True
```

Comment:
We need to find out how it will effect prod ?

Reply:
We currently have `USE_L10N = True` in our PRs. Since this setting is [enabled by default in Django 4.2](https://github.com/boxed/django/commit/9582f2f9d173935fef35c8c27b0f80258b1fb106) and [removed entirely in Django 5.2,](https://docs.djangoproject.com/en/5.2/releases/5.0/#:~:text=The%20USE_L10N%20setting%20is%20removed.) it is safe to remove without causing any issues in production.

P.S. As @irtazaakram  shared this URL, there is [no instance of `USE_L10N = False` in Open edX](https://github.com/search?q=org%3Aopenedx+%22USE_L10N+%3D+False%22&type=code), so removing it aligns with the existing usage across the platform.